ext/BreezeCloudMicrophysicsExt/one_moment_microphysics.jl

```diff
@@ -357,10 +357,9 @@ where the temperature derivative of saturation specific humidity is:
 ```
 
 # References
-- Morrison, H. and Milbrandt, J.A. (2015), J. Atmos. Sci., 72, 287-311.
-  https://doi.org/10.1175/JAS-D-14-0065.1
-- Morrison, H. and Grabowski, W.W. (2008), J. Atmos. Sci., 65, 1528-1548.
-  https://doi.org/10.1175/2007JAS2491.1
+* Morrison, H., and J. A. Milbrandt (2015). Parameterization of cloud microphysics based on
+    the prediction of bulk ice particle properties. Part I: Scheme description and idealized
+    tests. J. Atmos. Sci., 72, 287–311.
```

Comment:
Any reason for dropping one paper?  And the DOI was useful for getting a direct reference to the paper, instead of having to search it.

Reply:
The other paper wasn't mentioned in the docstring so I assumed it wasn't relevant?

Reply:
[2b1a6af](https://github.com/NumericalEarth/Breeze.jl/pull/345/commits/2b1a6af469936e8b0c85ec9f0b0b3635fcbabd00)